packages/maskbook/src/plugins/Snapshot/UI/SnapshotCard.tsx

```diff
@@ -4,24 +4,25 @@ import { useI18N } from '../../../utils/i18n-next-ui'
 const useStyles = makeStyles((theme) => {
     return createStyles({
         root: {
-            margin: '16px auto',
-            border: `solid 1px ${theme.palette.divider}`,
+            minHeight: 120,
             padding: 0,
-            minHeight: 320,
+            border: `solid 1px ${theme.palette.divider}`,
+            margin: 'theme.spacing(2) auto',
```

Comment:
You need string template here.

Reply:
wait so you leave this and merged?

cc @zhouhanseng @guanbinrui

Reply:
should I create a new PR to fix this?

Reply:
yes please